php_phongo.c

```diff
@@ -1561,11 +1553,6 @@ static void php_phongo_cursor_iterator_dtor(zend_object_iterator *iter TSRMLS_DC
 {
 	php_phongo_cursor_iterator *cursor_it = (php_phongo_cursor_iterator *)iter;
 
-	if (cursor_it->cursor->firstBatch) {
-		bson_clear(&cursor_it->cursor->firstBatch);
-		cursor_it->cursor->firstBatch = NULL;
-	}
-
 	php_phongo_cursor_free_current(cursor_it->cursor);
```

Comment:
Is this `php_phongo_cursor_free_current()` call actually necessary? `php_phongo_cursor_free()` already calls `php_phongo_cursor_free_current()`, and that will be invoked by `php_phongo_cursor_free_object()` when the zval is destroyed.